Translate remaining Chinese content in README.en.md

QilanLin opened the pull request.

sparanoid committed:
bump version to 0.0.3

summerblue committed:
add phphub to who-is-using section

sparanoid committed:
Merge pull request #14 from summerblue/patch-1

feat(docs): add phphub to who-is-using section

sparanoid committed:
chore: try flat badge (powered by shields.io)

sparanoid committed:
chore: rebrand my client LeanCloud (aka. AVOS Cloud)

sparanoid committed:
fix(docs): avoid personal writing style

sparanoid committed:
feat(docs): add new sections to Spacing

sparanoid committed:
fix(docs): typo

sparanoid committed:
chore: use original badge

sparanoid committed:
Merge branch 'master' of github.com:sparanoid/chinese-copywriting-guidelines

hotoo committed:
Add Vim plugin.

sparanoid committed:
Merge pull request #17 from hotoo/hotoo-patch-vim-plugin

feat(docs): add Vim plugin

sparanoid committed:
feat(docs): remove vanity PR tips

sparanoid committed:
feat(docs): more specific plugin description

sparanoid committed:
feat(docs): add usage for degrees and percentages

sparanoid committed:
wip: try `grunt-release`

sparanoid committed:
Merge branch 'master' of github.com:sparanoid/chinese-copywriting-guidelines

sparanoid committed:
chore: release 0.0.4

sparanoid committed:
feat(docs): avoid unidiomatic jargons

sparanoid committed:
chore: upgrade hardwares

sparanoid committed:
fix(docs): wrong heading

sparanoid committed:
chore: change description position

sparanoid committed:
fix(docs): typo HTML5, fixes https://github.com/sparanoid/chinese-copywriting-guidelines/issues/19

sparanoid committed:
chore: kill a weird character

sparanoid committed:
chore: bump deps version

sparanoid committed:
fix(package): license not defined

sparanoid committed:
chore: release 0.0.5

sparanoid committed:
chore: fix changelog

district10 committed:
add an empty line before quotation

sparanoid committed:
Merge pull request #20 from district10/master

chore: added an empty line before quotation

sparanoid committed:
wip(grunt): try `grunt-bump`

sparanoid committed:
feat(package): update contributors

sparanoid committed:
feat(grunt): better version handling

sparanoid committed:
chore: release v0.0.6

sparanoid committed:
feat(package): update contributors

nauxliu committed:
Update README.md

sparanoid committed:
feat(docs): add new tool

sparanoid committed:
Merge pull request #22 from NauxLiu/patch-1

添加 nauxliu/auto-correct

sparanoid committed:
chore(docs): update TOC

sparanoid committed:
feat(docs): add languages section

greenkeeperio-bot committed:
chore(package): update dependencies

http://greenkeeper.io/

sparanoid committed:
feat(docs): simplify TOC

sparanoid committed:
feat(docs): remove custom emojis

sparanoid committed:
feat(docs): simplify examples

sparanoid committed:
feat(docs): add badge for HTML version

sparanoid committed:
Merge pull request #24 from sparanoid/greenkeeper-update-all

Update all dependencies 🌴

sparanoid committed:
chore: release v0.0.7

sparanoid committed:
feat(package): update contributors

sparanoid committed:
fix(docs): avoid slangs

sparanoid committed:
feat(docs): add Python library from hjiang/scripts

sparanoid committed:
feat(docs): re-evaluate V2EX

sparanoid committed:
feat(docs): update spacing rule for links

sparanoid committed:
feat(docs): use original titles for references

sparanoid committed:
feat(docs): initial English version

greenkeeperio-bot committed:
chore(package): update grunt-bump to version 0.7.0

sparanoid committed:
Merge pull request #25 from sparanoid/greenkeeper-grunt-bump-0.7.0

Update grunt-bump to version 0.7.0 🚀

sparanoid committed:
fix(docs): wrong TOC anchors

sparanoid committed:
feat(docs): add language list

sparanoid committed:
chore(docs): add language list

sparanoid committed:
feat(package): update contributors

sparanoid committed:
chore: release v0.0.8

mzlogin committed:
fix(docs): wrong TOC anchors

sparanoid committed:
Merge pull request #26 from mzlogin/master

fix(docs): wrong TOC anchors

sparanoid committed:
refactor(docs): update TOC

greenkeeperio-bot committed:
chore(package): update matchdep to version 1.0.1

greenkeeperio-bot committed:
chore(package): update grunt-conventional-changelog to version 6.0.0

sparanoid committed:
Merge pull request #28 from sparanoid/greenkeeper-grunt-conventional-changelog-6.0.0

Update grunt-conventional-changelog to version 6.0.0 🚀

greenkeeperio-bot committed:
chore(package): update grunt-conventional-changelog to version 6.0.1

sparanoid committed:
Merge pull request #29 from sparanoid/greenkeeper-grunt-conventional-changelog-6.0.1

grunt-conventional-changelog@6.0.1 untested ⚠️

greenkeeperio-bot committed:
chore(package): update grunt-conventional-changelog to version 6.1.0

sparanoid committed:
Merge pull request #30 from sparanoid/greenkeeper-grunt-conventional-changelog-6.1.0

grunt-conventional-changelog@6.1.0 untested ⚠️

ddanist committed:
一些标点符号的更正。

sparanoid committed:
Merge pull request #27 from sparanoid/greenkeeper-matchdep-1.0.1

matchdep@1.0.1 untested ⚠️

sparanoid committed:
Merge pull request #31 from ddanist/master

feat(docs): wrong quote marks usage

sparanoid committed:
feat(docs): update repo link

sparanoid committed:
feat(docs): update link redirects, props @ReadmeCritic

sparanoid committed:
chore: release v0.0.9

sparanoid committed:
feat(package): update contributors

mzlogin committed:
feat(docs): update punctuations

sparanoid committed:
Merge pull request #35 from mzlogin/master

chore(docs): sync Chinese version change to English version

greenkeeperio-bot committed:
Update grunt to version 1.0.1 🚀

greenkeeperio-bot committed:
chore(package): update grunt-bump to version 0.8.0 (#40)

huhuhang committed:
Update README.md (#46)

* Update README.md

「誰在這樣做？」小節增加少數派 http://sspai.com

* Update README.md

mzlogin committed:
fix(docs): update adverb

greenkeeper[bot] committed:
docs(readme): add Greenkeeper badge

pityonline committed:
feat(docs): add another bad example to spacing

Sometimes, additional spaces are added before/after punctuation in fullwidth,
especially in copying Chinese translation in Google Translate. It's better to
avoid spacing in this situation, or it looks wider than normal.

jxlwqq committed:
feat(docs): add jxlwqq/chinese-typesetting (#62)

該倉庫有更加豐富，完整的功能。

sparanoid committed:
chore: update deps

sparanoid committed:
feat(docs): add python lib hint, fixes https://github.com/sparanoid/chinese-copywriting-guidelines/issues/50

sparanoid committed:
feat(docs): examples cleanup

sparanoid committed:
chore(docs): add external link indicator

sparanoid committed:
fix(docs): wrong external link indicator

sparanoid committed:
chore(docs): use localized string

sparanoid committed:
feat(docs): add `text spacing` notice, fixes https://github.com/sparanoid/chinese-copywriting-guidelines/issues/51

sparanoid committed:
feat(package): update contributors

sparanoid committed:
chore: release v0.0.10

sparanoid committed:
feat(docs): switch license to MIT to meet OSI reqirements

sparanoid committed:
fix(grunt): duplicated default task

sparanoid committed:
chore: release v0.1.0

sparanoid committed:
Update Crowdin configuration file

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (Chinese Traditional, Hong Kong)

sparanoid committed:
New translations README.md (English)

sparanoid committed:
New translations README.md (English)

sparanoid committed:
feat(docs): add badge for Crowdin

sparanoid committed:
New translations README.md (English)

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (Chinese Traditional, Hong Kong)

sparanoid committed:
feat(docs): avoid hardcoded TOC

sparanoid committed:
feat(docs): update language links

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (Chinese Traditional, Hong Kong)

sparanoid committed:
New translations README.md (English)

sparanoid committed:
New translations README.md (English)

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (Chinese Simplified)

Kuri-su committed:
docs(readme): update Readme

Kuri-su committed:
Update README.md

renovate-bot committed:
Add renovate.json

sparanoid committed:
chore(renovate): reserve semver

polaris1119 committed:
Update README.md

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (Chinese Traditional, Hong Kong)

sparanoid committed:
New translations README.md (English)

sparanoid committed:
New translations README.md (English)

sparanoid committed:
New translations README.md (English)

sheiun committed:
Update README.md

Translate README.md from Chinese Simplified to Chinese Traditional.

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (Chinese Traditional, Hong Kong)

sparanoid committed:
New translations README.md (English)

sparanoid committed:
New translations README.md (English)

sparanoid committed:
fix: typo

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (Chinese Traditional, Hong Kong)

sparanoid committed:
New translations README.md (English)

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (English)

sparanoid committed:
chore: enforce https for links

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (Chinese Traditional, Hong Kong)

sparanoid committed:
New translations README.md (English)

sparanoid committed:
chore: add FUNDING.yml

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
chore: typo

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (Chinese Traditional, Hong Kong)

sparanoid committed:
New translations README.md (English)

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New translations README.md (English)

sparanoid committed:
New translations README.md (English)

sparanoid committed:
chore: remove deprecated english version

zry656565 committed:
fix: add leetcode to the list

sparanoid committed:
New translations README.md (Chinese Simplified)

sparanoid committed:
New Crowdin translations (#97)

New Crowdin translations

reilee committed:
Update README.zh-CN.md

贾伯斯 -> 乔布斯

n0vad3v committed:
Add n0vad3v/Tekorrect to list

sparanoid committed:
New translations README.md (English)

cshung1994 committed:
Correct some words

liyishuai committed:
英文书籍名以斜体表示

louis70109 committed:
Update README.md

sparanoid committed:
New translations README.md (Chinese Traditional, Hong Kong)

hms5232 committed:
Add quote block to README

使用 Markdown 的引言語法

huacnlee committed:
Add Rust, Go version. Add VS Code extension link.

sparanoid committed:
Merge pull request #131 from huacnlee/add-tool

sparanoid committed:
feat: proofreading en-US

Co-authored-by: Yishuai Li <liyishuai@users.noreply.github.com>

sparanoid committed:
Merge pull request #105 from sparanoid/l10n-master

sparanoid committed:
chore: sync from master

sparanoid committed:
Merge branch 'master' into master

sparanoid committed:
Merge pull request #129 from hms5232/master

sparanoid committed:
chore: keep bracket quote marks usage

sparanoid committed:
Merge pull request #126 from louis70109/patch-1

sparanoid committed:
chore: keep title navigating to the north

sparanoid committed:
Merge pull request #106 from cshung1994/patch-2

sparanoid committed:
Merge pull request #94 from zry656565/add-leetcode-to-list

sparanoid committed:
Merge branch 'master' into patch-1

sparanoid committed:
Merge pull request #101 from reilee/patch-1

sparanoid committed:
Merge branch 'master' into master

sparanoid committed:
Merge pull request #103 from n0vad3v/master

sparanoid committed:
feat: merge zh-TW and zh-HK into default (zh-Hant)

sparanoid committed:
feat: migrate Chinese Simplified and English

sparanoid committed:
chore: update badges

sparanoid committed:
fix: broken links

sparanoid committed:
refactor: simplify tables

sparanoid committed:
feat: unify license

sparanoid committed:
feat: keep language switcher in English

sparanoid committed:
feat: remove third-party deps monitor

sparanoid committed:
feat: remove old-fashioned tech

sparanoid committed:
chore: init release-it

sparanoid committed:
chore: release v1.0.0

sparanoid committed:
fix: use term article for wikipedia, fix https://github.com/sparanoid/chinese-copywriting-guidelines/issues/122

sparanoid committed:
fix: wrong term usage in zh-Hans, fix https://github.com/sparanoid/chinese-copywriting-guidelines/issues/99

sparanoid committed:
fix: outdated product name copywriting, fix https://github.com/sparanoid/chinese-copywriting-guidelines/issues/93

sparanoid committed:
feat: remove table of contents placeholder

louis70109 committed:
Update README.md

- Modify 存儲 to 儲存

yslinear committed:
fix: correct some words

sparanoid committed:
Merge pull request #164 from yslinear/master

fix: correct some words

sparanoid committed:
Merge pull request #154 from louis70109/patch-1

Update README.md

sparanoid committed:
chore: remove dead badge service

sparanoid committed:
chore: sync examples from upstream

sparanoid committed:
chore: redundant badges

sparanoid committed:
chore: remove package placeholder, cannot find an easy way to update contributers :(

sparanoid committed:
feat: sync and remove sites no longer meet requirements

sparanoid committed:
feat: use chinese simplified for specific examples. fix https://github.com/sparanoid/chinese-copywriting-guidelines/issues/151

sparanoid committed:
feat: refine examples of degrees, fix https://github.com/sparanoid/chinese-copywriting-guidelines/issues/161

sparanoid committed:
feat: move forked projects, fix https://github.com/sparanoid/chinese-copywriting-guidelines/issues/152

sparanoid committed:
feat: add basic markdown lint

sparanoid committed:
feat: add basic github action

sparanoid committed:
chore: release v1.0.1

yslinear committed:
fix: correct some words

sparanoid committed:
Merge pull request #167 from yslinear/master

fix: 移除多餘的星號

serkodev committed:
docs: add vue-pangu tool

sparanoid committed:
Merge pull request #168 from serkodev/master

HeavySnowJakarta committed:
Update README.md

据 comment#165 討論結果進行修訂

HeavySnowJakarta committed:
Update README.zh-Hans.md

据 comment#165 讨论结果进行修订

HeavySnowJakarta committed:
Update README.en.md

Revise according to the result of comment#165

sparanoid committed:
Merge pull request #171 from HeavySnowJakarta/patch-1

huacnlee committed:
Update Tool list to add new tools and remove no maintained projects.

QilanLin committed:
Update README.en.md

translate some Chinese content

sparanoid committed:
Update README.en.md

Co-authored-by: Yishuai Li <liyishuai.lys@alibaba-inc.com>

sparanoid committed:
Update README.en.md

Co-authored-by: Yishuai Li <liyishuai.lys@alibaba-inc.com>

sparanoid committed:
Merge pull request #183 from QilanLin/master

sparanoid committed:
Merge pull request #182 from huacnlee/improve-tool-list

Commit:
Fix #190

sparanoid committed:
Merge pull request #191 from liyishuai/fix-190

sparanoid committed:
Merge pull request #115 from liyishuai/master

sparanoid committed:
feat: switch to pnpm

sparanoid committed:
Merge pull request #193 from sparanoid/pnpm

sparanoid committed:
feat: use renovate for deps monitoring

sparanoid committed:
chore: no funding needed for this project

renovate[bot] committed:
chore(deps): update dependency remark-cli to v11

sparanoid committed:
chore: update license

sparanoid committed:
Merge pull request #194 from sparanoid/renovate/major-remark-monorepo

LiLittleCat committed:
feat: add Pangu series Intellij platform plugin to tools

sparanoid committed:
Merge pull request #197 from LiLittleCat/master

lomirus committed:
Add missing spaces

sparanoid committed:
Merge pull request #198 from lomirus/master

QilanLin committed:
Update README.en.md

liyishuai commented:
My comment remains the same as https://github.com/sparanoid/chinese-copywriting-guidelines/pull/185#discussion_r1116424862. Feel free to pick or drop them.

QilanLin commented:
<img width="825" alt="Screenshot 2023-08-31 at 13 36 54" src="https://github.com/sparanoid/chinese-copywriting-guidelines/assets/78576883/db45ac78-98c1-441c-b380-a8b427ea6c11">

QilanLin commented:
<img width="825" alt="Screenshot 2023-08-31 at 13 36 54" src="https://github.com/sparanoid/chinese-copywriting-guidelines/assets/78576883/969ea5e9-7a6f-4bda-9fa1-64d25237e817">

liyishuai commented:
I have no privilege to decide whether to merge your PR or not, and have no privilege to modify your fork.
In other words, what you should do is to convince sparanoid---the repository owner---instead of convincing GPT or educating me. My being educated or not does not increase or decrease the chance of this PR being merged.

QilanLin commented:
> My comment remains the same as https://github.com/sparanoid/chinese-copywriting-guidelines/pull/185#discussion_r1116424862. Feel free to pick or drop them.

> what you should do is to convince sparanoid---the repository owner---instead of convincing GPT or educating me. 

It was only a response in the same way you did in https://github.com/sparanoid/chinese-copywriting-guidelines/pull/185#discussion_r1116424862, where you used GPT to educate (if u think so) me as well.

Regarding your comment in #185, please consider what GPT-4 suggests:

"Easy to read"（易于阅读）确实是文本翻译中一个重要的考量因素，但它并不应该牺牲准确性和清晰性。在这个具体例子中，第二个翻译不仅准确而且清晰，而且并没有特别难以理解或阅读。因此，在这种情况下，追求“易于阅读”并不意味着你应该选择第一个翻译。

如果你觉得第二个翻译过于复杂或冗长，那么最好的做法是尝试在保持准确性和清晰性的前提下简化它，而不是转而选择一个可能更易阅读但较不准确或不清晰的翻译。

我的观点是，除非第二个翻译真的特别难以理解（这在这里并不是问题），否则没有必要用第一个翻译来替换它。

liyishuai commented:
很好，我们的讨论终于回到内容本身，而非谁改谁这种行为问题上了。

我的修辞准则，以供参考：
1. 谓语是句子的核心。如果一句话的主题是大小写，那么谓语取capitalize比write更有利于读者抓住重点。
2. 后置从句只用于补充说明。如果HTML是重要的论域，则应前置约束。

QilanLin committed:
Update README.en.md

“用标准的大小写规范进行书写” 翻译为 capitalize 不太准确

QilanLin commented:
input 做动词应该会更准确

liyishuai commented:
input的意思是“吞”数据，是电脑的行为。
建议是向开发者提出的。开发者的行为是“敲”键盘、“编码”，故建议改用type、encode等动词。

QilanLin committed:
remove the redundant "n"

QilanLin committed:
ibid

QilanLin committed:
remove the redundant "n"

QilanLin committed:
Update README.en.md

QilanLin committed:
Update README.en.md

QilanLin committed:
Update README.en.md

GPT-4:
从语法角度来看，第二句话 "Exception: inline English titles of books and journals with italics, instead of fullwidth title marks." 是正确的。在这一句中，"Exception:" 作为一个项目或规则的特殊情况的标签，并后接英文的解释，整个句子逻辑清晰、语法准确。

第一句话中的 "Exceptions: inline English titles of books and journals with italics, instead of fullwidth title marks." 使用了复数形式的 "Exceptions"，而其后的描述实际上是针对一个特定的例外情况，因此使用单数 "Exception" 更准确。

需要注意的是，这里我是从严格的语法和句子结构出发进行解释的。在实际应用中，有时候人们可能会灵活使用 "Exceptions" 和 "Exception"，但从规范的角度，第二句更为准确。

liyishuai reviewed (commented):
LGTM. Seems ready to merge.
